Makefile

```diff
@@ -19,7 +19,7 @@ clean::  ## Remove generated files
 
 # --- Build --------------------------------------------------------------------
 GO_LDFLAGS = -X main.version=$(VERSION)
-CMDS = .
+CMDS = . ./pb/client ./pb/server
```

Comment:
I have a slight discomfort at non-generated code being in `pb` and that `make protos` is going to run `gofumpt` over them too. I wonder if perhaps we put it in `internal/client` and `internal/server`? or just in `cmd` like usual? we can still keep `main.go` at the top so that `go install foxygo.at/pony` works.

Reply:
ok. i think internal is fine. client and server are a bit generic terms anyway. done.